data/bboxes.js

```diff
@@ -0,0 +1,2954 @@
+var bboxes = {
```

Comment:
Suppose it's not a huge deal right now, but come redistricting it may start to be a pain.  Especially with the court challenges.